build/commands/lib/config.js

```diff
@@ -177,6 +176,35 @@ Config.prototype.isOfficialBuild = function () {
   return this.buildConfig === 'Release'
 }
 
+Config.prototype.isStableChannel = function () {
+  return this.channel === '' || this.channel === 'release'
+}
+
+Config.prototype.isBetaChannel = function () {
+  return this.channel === 'beta'
+}
+
+Config.prototype.isNightlyChannel = function () {
+  return this.channel === 'nightly'
+}
+
+Config.prototype.isDevChannel = function () {
+  return this.channel === 'dev' || this.channel === 'development'
```

Comment:
`dev` and `development` are not the same thing. `dev` and `beta` are currently equivalent in terms of releases and `development` is local only

Reply:
Got it, thanks.
Removed this functions.